lib/tests/streamlit/runtime/runtime_test.py

```diff
@@ -58,7 +58,7 @@ async def test_start_stop(self):
         """starting and stopping the Runtime should work as expected."""
         self.assertEqual(RuntimeState.INITIAL, self.runtime.state)
 
-        await self.start_runtime_loop()
+        await self.runtime.start()
```

Comment:
In our tests, we can now just call `Runtime.start` (or `Server.start`) directly, rather than relying on a custom testing method.